tests/cases/fourslash/completionForStringLiteralExport.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference path='fourslash.ts' />
+
+// Should define spans for replacement that appear after the last directory seperator in export statements
+
+// @typeRoots: my_typings
+
+// @Filename: test.ts
+
+//// export * from  "./[|some|]/*0*/
+//// export * from  "./sub/[|some|]/*1*/";
+//// export * from  "[|some-|]/*2*/";
+//// export * from  "../[||]/*3*/";
+//// export {} from  "./[|some|]/*4*/";
```

Comment:
Do you need the range markers here? If it's not too much trouble, could you remove them?

Reply:
sure😂